contracts/ownership/Whitelist.sol

```diff
@@ -11,84 +11,88 @@ import "./rbac/RBAC.sol";
  * @dev This simplifies the implementation of "user permissions".
  */
 contract Whitelist is Ownable, RBAC {
-  event WhitelistedAddressAdded(address addr);
-  event WhitelistedAddressRemoved(address addr);
```

Comment:
Why to remove these?

Reply:
The RoleAdded/Removed events cover this use case :)